src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/VisualRulesTab.tsx

```diff
@@ -1,6 +1,96 @@
-import React from 'react';
+import { Stack, useTheme, Text, ActionButton } from '@fluentui/react';
+import React, { useCallback, useState } from 'react';
+import { useBoolean } from '@fluentui/react-hooks';
+
+import { useTranslation } from 'react-i18next';
+import {
+    getActionButtonStyles,
+    getLeftPanelStyles
+} from '../../Shared/LeftPanel.styles';
 import { VisualRulesList } from '../VisualRules/VisualRuleList';
+import { IVisualRule } from '../VisualRules/VisualRules.types';
 
+const ROOT_LOC = '3dSceneBuilder.behaviorVisualRulesForm';
```

Comment:
nit: I would change this to '3dSceneBuilder.visualRulesTab', this is technically not a form. If there is already a '3dSceneBuilder.visualRulesTab' value, I would include it in this object ('3dSceneBuilder.visualRulesTab.tabName').